.github/CODEOWNERS

```
@@ -5,9 +5,9 @@
 
 # Secondary repo maintainers, substitutes of the primary
 # maintainers when they become MIA
-*       @cwgoes @sunnya97
+*       @cwgoes @marbar3778 @ebuchman @robert-zaremba @amaurym @fedekunze @odeke-em
 
 # Primary repo maintainers
-*       @aaronc @alexanderbez @alessio
+*       @aaronc @alexanderbez @alessio @zmanian @sunnya97
```

```suggestion
*       @aaronc @alexanderbez @zmanian @sunnya97
```

I need to elaborate on this. Yesterday [I resigned from my position as a member of the Concept Approval committee](https://gist.githubusercontent.com/alessio/a7212490f9d2f4c44c5e2f59b9e0052b/raw/6c5575bb37c18a895cdb150e3e1abd49f5fea4c0/cosmos-adr-commitee-resignation.asc). I've delivered my resignation to both @aaronc, @alexanderbez, and Interchain Foundation. Please note that this is **NOT** an attempt to escalate any issue whatsoever further.

I've had a pleasant conversation with @okwme and @hxrts, at the end of which we mutually agreed this was the best course of action in the interest of working more closely and smoothly with Tendermint going forward - not less. In particular, I want to thank @hxrts for the nice words he had for me. I appreciated that and promised to step down responsibly and do my best to ease the transition.

Ad maiora!